Data/core_rulebook/cr_abilities.lua

```diff
@@ -9133,6 +9298,10 @@ DefineAbility({
   Category="Special Ability",
   AllowMultiple=true,
   Stackable=true,
+  Choice={
```

Comment:
What are the other keys for "Choice"? other than "Choose"?

Reply:
MaxTimes